compact/fields/beamline_10x100_Au197.xml

```diff
@@ -0,0 +1,68 @@
+<!-- SPDX-License-Identifier: LGPL-3.0-or-later -->
+<!-- Copyright (C) 2023 Sylvester Joosten, Alex Jentsch, Whitney Armstrong, Simon Gardner -->
+
+  <define>
+
+    <constant name="ElectronBeamEnergy" value="10*GeV"  />
+    <constant name="IonBeamEnergy"      value="115*GeV" />
```

Comment:
Should this be
```suggestion
    <constant name="IonBeamEnergy"      value="100*GeV" />
```

Reply:
Yes - sorry, I made that change but forgot to push that commit.